util/buildRelease/gen_release

```diff
@@ -60,7 +60,7 @@ if (exists($ENV{"CHPL_GEN_RELEASE_NO_CLONE"})) {
 
     # copy the current CHPL_HOME into the directory where chapel will be built
     print "Creating build workspace with tar...\n";
-    SystemOrDie("cd $chplhome && tar -cf - . | (cd $archive_dir && tar -xf -)");
+    SystemOrDie("cd $chplhome && tar -cf - . | (cd $archive_dir && tar -xf - --exclude .gitignore --exclude .gitattributes)");
```

Comment:
This works -  I would suggest one of the following improvements for maintainability:

1) remove the files in a separate step, like we do for other files we remove in this script

2) bundle the excluded files into a variable including prepended `--exclude `s that we just pass to the tar command.